backend/api/serializers/computetask.py

```diff
@@ -232,6 +226,41 @@ def _replace_storage_addresses(self, task):
                 reverse("api:function-file", args=[task["function"]["key"]])
             )
 
+
+class ComputeTaskWithDetailsSerializer(ComputeTaskSerializer):
+    inputs = ComputeTaskInputSerializer(many=True)
+    outputs = ComputeTaskOutputSerializer(many=True)
+
+    class Meta:
+        model = ComputeTask
+        fields = [
+            "function",
+            "channel",
+            "compute_plan_key",
+            "creation_date",
+            "end_date",
+            "error_type",
+            "key",
+            "logs_permission",
+            "metadata",
+            "owner",
+            "rank",
+            "start_date",
+            "status",
+            "tag",
+            "worker",
+            "duration",
+            "inputs",
+            "outputs",
+        ]
```

Comment:
Maybe you can re-use the fields in the other Serializer ?
```suggestion
        fields = ComputeTaskSerializer.Meta.fields + [
            "inputs",
            "outputs",
        ]
```